lambdatimes/src/components/Content/Content.js

```diff
@@ -37,7 +49,12 @@ export default class Content extends Component {
           of the items from cardData. 
         - else, it should only return those cards whose 'tab' matched this.state.selected.
     */
-    return this.state.cards;
+   
+    if(this.state.selected === 'all'){
+      return this.state.cards;
+    }else{
+      return this.state.cards.filter(card => card.tab === this.state.selected);
+    }
```

Comment:
You could probably make this a bit cleaner with destructuring. For example:
```javascript
const { cards, selected } = this.state;
if (selected === 'all'){
  return cards;
} else {
  return cards.filter(({ tab }) => tab === selected);
}
```
or, with a ternary:
```javascript
const { cards, selected } = this.state;
return selected === 'all'
  ? cards
  : cards.filter(({ tab }) => tab === selected);
```